recipes/quazip/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir build
+
+cmake -B build -S . -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
maybe you could use `${CMAKE_ARGS}`? That would also help cross-compiling in the future. 
You can also do that for the Windows build file.